lib/constants/enums.ts

```diff
@@ -14,22 +14,50 @@ export enum OrderSide {
   Sell,
 }
 
-export enum Network {
+export enum XUNetwork {
```

Comment:
I'll change it to `XuNetwork`. I wanted to refrain from `Network` because 1) we have a similar name object and 2) it might be too generic for easily understanding that it's about an Exchange Union network.